app/pages/_app.js

```diff
@@ -0,0 +1,46 @@
+import { createGlobalStyle, ThemeProvider } from 'styled-components';
+import { Footer, Navigation } from '@button-inc/bcgov-theme';
+import styled from 'styled-components';
+import Menu from '../components/Menu';
+import BCGovTypography from '../components/BCGovTypography';
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+`
+
+const theme = {
+  colors: {
+    primary: '#0070f3',
+  },
+};
+
+const SFooter = styled(Footer)`
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+`;
+
+export default function App({ Component, pageProps }) {
+  return (
+    <>
+      <GlobalStyle />
+      <ThemeProvider theme={theme}>
+        <BCGovTypography />
+        <Navigation
+          header="main"
+          title="Connectivity Intake"
+        >
+          <Menu />
+        </Navigation>
+        <Component {...pageProps} />
```

Comment:
In [the Demo](https://button-inc.github.io/service-development-toolkit/@button-inc/bcgov-theme/index.html?path=/docs/getting-started-toolkit-in-action--default) we see the page layout has a `2rem` padding around the main content. Let's shoot for consistency here?

```
  <div
    style={{
      padding: '2rem'
    }}
  >
  ```